src/data/scripts/merge_jsonl/README.md

```diff
@@ -0,0 +1,9 @@
+# JSONL File Merger
+
+This Python script merges JSONL (JSON Lines) files from a specified folder into a single output file.
```

Comment:
Where is `## Dependencies` ex. `pip install ...`?

Reply:
done